web/components/shared/themed/themedAdvancedFilters.tsx

```diff
@@ -28,6 +30,9 @@ export function AdvancedFilters({
     property: string,
     search: string
   ) => Promise<Result<void, string>>;
+  filterLayouts?: {
```

Comment:
nit: Are we calling saved filters layouts? we should keep the naming consistent. `Saved Filters` makes sense to me as opposed to layouts, but im cool either way